README.md

```diff
@@ -229,7 +229,7 @@ This design decision does not affect the performance of queries.
 
 **By the way, you are able to use SeaGOAT to query your repository while
 it's processing your files!** When you make a query, and the files are not
-processed yet, you will receive a warning with an esimation of the accuracy
+processed yet, you will receive a warning with an estimation of the accuracy
```

Comment:
are you referring to this change to read the comments?  Please clarify what do you want me to do.